src/app/ipv/router.js

```diff
@@ -15,42 +15,33 @@ const {
   handleAppStoreRedirect,
 } = require("./middleware");
 
-// Remove this as part of PYIC-4278
-const { allTemplatesMoved } = require("../development/middleware");
-const { getRoutePath } = require("../../lib/paths");
-const path = require("path");
-const { UPDATE_DETAILS } = require("../../constants/ipv-pages");
+const {
+  PYI_ATTEMPT_RECOVERY,
+  UPDATE_DETAILS,
+} = require("../../constants/ipv-pages");
 
 const csrfProtection = csrf({});
 const parseForm = bodyParser.urlencoded({ extended: false });
 
-router.get(
-  path.join("/", "usefeatureset"),
-  validateFeatureSet,
-  renderFeatureSetPage,
-);
+function getPagePath(pageId) {
+  return `/page/${pageId}`;
+}
+
+router.get("/usefeatureset", validateFeatureSet, renderFeatureSetPage);
 
 router.get(
-  path.join("/", "page", "attempt-recovery"),
+  getPagePath(PYI_ATTEMPT_RECOVERY),
```

Comment:
This is a change from `attempt-recovery` to `pyi-attempt-recovery`.

I can't see any evidence that we use `attempt-recovery` (without the PYI prefix).

Reply:
The page url was `attempt-recovery` before-hand, so I wonder whether it's ever used?
```
router.get("/page/attempt-recovery", csrfProtection, renderAttemptRecoveryPage);
```

Reply:
I suspect you're right, although this might save an extra (pointless) redirect, if it's ever possible to be directed to `pyi-attempt-recovery` directly...

I'm tempted to leave it?